infrastructure/kube/templates/relay/initcontainer/provision-relay/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:13-alpine AS install
+
+RUN apk --no-cache add git
+
+WORKDIR /tmp
+
+COPY ./package.json /tmp/package.json
+COPY ./package-lock.json /tmp/package-lock.json
+
+RUN npm ci
+
+FROM node:13-alpine
+
+WORKDIR /tmp
+
+COPY --from=install /tmp .
```

Comment:
Is it worth using the `install` aliased image and copying the contents of the whole `/tmp` directory? What are we trying to achieve here? We will copy the whole `node_modules` anyway, so will the resulting image be any lighter?

Reply:
It's taken from the legacy relay maintainer `Dockerfile` and it's taken because`node:13-alpine` gives a smaller image, though it requires a multistage build since the base Alpine image doesn't include git. (reference commit: https://github.com/keep-network/tbtc/commit/418c6e200b8c321e120d65fbd562d1a65d350828)